Libraries/CustomComponents/Navigator/Navigator.js

```diff
@@ -139,26 +139,80 @@ var GESTURE_ACTIONS = [
  * ### Basic Usage
  *
  * ```
- *   <Navigator
- *     initialRoute={{name: 'My First Scene', index: 0}}
- *     renderScene={(route, navigator) =>
- *       <MySceneComponent
- *         name={route.name}
- *         onForward={() => {
- *           var nextIndex = route.index + 1;
- *           navigator.push({
- *             name: 'Scene ' + nextIndex,
- *             index: nextIndex,
- *           });
- *         }}
- *         onBack={() => {
- *           if (route.index > 0) {
- *             navigator.pop();
- *           }
- *         }}
- *       />
+ *   // NavigatorExample.js
+ *
+ *   'use strict';
+ *
+ *   var React = require('react-native');
+ *   var {
+ *     Navigator,
+ *     AppRegistry,
+ *     StyleSheet,
+ *     Text,
+ *     View,
+ *   } = React;
+ *
+ *   var SomeScene = React.createClass({
+ *     onForward() {
+ *       var nextIndex = this.props.index + 1;
+ *       this.props.navigator.push({
+ *         message: 'Scene ' + nextIndex,
+ *         index: nextIndex,
+ *       });
+ *     },
+ *
+ *     onBackward() {
+ *       this.props.navigator.pop();
+ *     },
+ *
+ *     render() {
+ *       return (
+ *         <View>
+ *           <Text onPress={this.onForward}>
+ *             {this.props.message}
+ *           </Text>
+ *
+ *           <Text onPress={this.onBackward}>
+ *             Press me to go back a scene.
+ *           </Text>
+ *         </View>
+ *       );
  *     }
- *   />
+ *   });
+ *
+ *   var NavigationExample = React.createClass({
+ *     renderScene(route, navigator) {
+ *       switch(route.message) {
+ *         // Cases for your various scenes go here
```

Comment:
I'd rather see this switch on something like route.name, instead of message. I would also remove the default route and instead explicitly show how that is matched in the renderScene method.